src/Components/NewForm.vue

```diff
@@ -35,6 +35,31 @@
 			rows="4"
 			:placeholder="t('announcementcenter', 'Write announcement text, Markdown can be used …')" />
 
+		<div class="announcement__form__schedule">
```

Comment:
In the meantime a design pattern for date pickers in dropdowns was added, so this could go into the ... menu we already have, to be less present for the default experience?
![grafik](https://github.com/user-attachments/assets/29401d5a-2f55-4384-8047-d6b914fdb356)

Reply:
But we can also iterate this in a follow up

Reply:
Moved to #820